addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7434,6 +7434,11 @@ registry.ImageTools = ImageHandlerOption.extend({
      */
     _computeWidgetState(methodName, params) {
         switch (methodName) {
+            case "selectAttribute":
+                if (params.attributeName === "alt" && params.activeValue.trim() !== "") {
+                    this.$target[0].removeAttribute("aria-hidden");
+                }
+                break;
             case 'selectStyle': {
```

Comment:
Not exactly sure why you want to remove the `"aria-hidden"` here ? But if you just want images with "non-empty alt" to be visible in the accessibility API, I think you also need to set it back when the `alt` is removed from the `Alt tag` option, no ?

Also, isn't better to do this in a `selectAttribute()` override instead of `_computeWidgetState()` ?